components/menu-items/toggle.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * Internal dependencies
+ */
+import MenuItemsItem from './item';
+import './style.scss';
+
+/**
+ * Renders the more menu toggle.
```

Comment:
Might be more explanatory if it said "Renders a toggle menu item for use inside the more menu."

Reply:
I will update, copied and updated from the referenced PR to be honest :)